includes/wikia/services/InspectletService.class.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Class InspectletService
+ * This service provides interface to add Inspectlet integrations to parts of the product
+ */
+class InspectletService extends WikiaService {
+	const COMMUNITY_PAGE = 'CommunityPage';
+	const CREATE_NEW_WIKI = 'CreateNewWiki';
+
+	private static $applicationIds = [
+		self::CREATE_NEW_WIKI => 55883171,
+		// @see https://wikia-inc.atlassian.net/browse/WW-111
+		self::COMMUNITY_PAGE => 1280339383
+	];
+
+	private $inspectletExperimentId;
+
+	/**
+	 * @param $pageName
+	 */
+	public function __construct( $pageName ) {
+		if ( !empty( static::$applicationIds[$pageName] ) ) {
+			$this->inspectletExperimentId = static::$applicationIds[$pageName];
+		} else {
+			throw new RuntimeException( "$pageName is not configured in InspectletService" );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS112)